src/components/generic/ActiveTooltip.tsx

```
@@ -0,0 +1,24 @@
+}: {
+	readonly t?: ReturnType<typeof useTranslations>;
+	readonly tooltipKey?: string;
+}) {
+	return (
+		<TooltipProvider>
+			<Tooltip>
+				<TooltipTrigger>
+					<CircleHelp className="w-4 h-4 cursor-pointer" />
+				</TooltipTrigger>
+				<TooltipContent>
+					<p dangerouslySetInnerHTML={{ __html: t?.(tooltipKey) || "" }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)